about.html

```diff
@@ -63,7 +63,7 @@ <h1 class="section__heading text-light-orange mb-3">Artificial Intelligence<br>G
       </div>
       <div class="d-flex flex-column justify-content-center align-items-center text-center">
         <p>Please contact us by Email for any further questions about Global AI Summit 2022</p>
-        <a class="text-decoration-none hover_underline hover_color_dark_black text-dark-black fw-bold" href="#">Info@GlobalAISummit.org</a>
+        <a class="text-decoration-none hover_underline hover_color_dark_black text-dark-black fw-bold" href="mailto: Info@GlobalAISummit.org">Info@GlobalAISummit.org</a>
```

Comment:
- [ ] `required`
- [ ] Amazing work you have done so far 👏 
- [ ] however, the `hamburger` is supposed to be on the left side not on the right side,.
- [ ] also the logo is not supposed to show in the mobile version as well as the first `navbar`
- [ ] Kindly make adjustments
- [ ] please see screenshots below

`Behance template`
![image](https://user-images.githubusercontent.com/66230306/196944147-a0377aa9-edd7-4d65-acba-bb7edfb05dbb.png)

`your template`
![image](https://user-images.githubusercontent.com/66230306/196945316-0ce843ed-4b23-4677-9a39-5bccf77c86c2.png)